realm/realm-library/src/androidTest/java/io/realm/ManagedOrderedRealmCollectionTests.java

```diff
@@ -113,6 +114,7 @@
  */
 
 @RunWith(Parameterized.class)
+@Ignore("Tests crash due to bug in core, see https://jira.mongodb.org/browse/RCORE-435")
```

Comment:
Is this still a problem? At least the comments in the issue seems to indicate it has been fixed?

If it hasn't been fixed, how critical is it?

Reply:
It has been fixed but it is on the core develop branch.

Reply:
Okay, in that case, I would put this PR on hold until we can get a fix from Core. But otherwise I would say this PR is ready to be merge.